Extensions/Spine/JsExtension.js

```diff
@@ -73,6 +73,42 @@ module.exports = {
       .setFunctionName('setAnimationMixingDuration')
       .setGetter('getAnimationMixingDuration');
 
+    object
+      .addExpressionAndCondition(
+        'number',
+        'PointAttachmentX',
+        _('Point attachment x'),
+        _('x position of spine point attachment'),
+        _(
+          'x position of spine _PARAM1_ point attachment for _PARAM2_ slot (use "" if names are the same)'
```

Comment:
> (use "" if names are the same)

What does it mean?
The value is a number, I'm not sure a double quote works there.